server/message_hooks.go

```diff
@@ -554,9 +554,10 @@ func (p *Plugin) handlePromptForConnection(userID, channelID string) {
 	}
 
 	if time.Until(timestamp) < -time.Hour*time.Duration(promptInterval) {
-		p.sendBotEphemeralPost(userID, channelID, "Your Mattermost account is not connected to MS Teams so your activity will not be relayed to users on MS Teams. You can connect your account using the `/msteams connect` slash command.")
-
-		if err = p.store.StoreDMAndGMChannelPromptTime(channelID, userID, time.Now()); err != nil {
+		connectURL := p.GetURL() + "/reconnect"
+		message := fmt.Sprintf("[Click here to re-connect your account](%s)", connectURL)
```

Comment:
I had to double check, but reconnect doesn't need a hyphen
```suggestion
		message := fmt.Sprintf("[Click here to reconnect your account](%s).", connectURL)
```